### DISCUSSION TITLE
有没有出合约开发框架系列的教程的规划？

### DISCUSSION THREAD

**Discussion by czp-first** [AmazingAng/WTF-Solidity#323] [General]
比如hardhat、truffle之类的

**Comment by AmazingAng** [comment]
有hardhat和Truffle的教程

https://github.com/AmazingAng/WTF-Solidity/tree/main/Topics/Tools

**Reply by czp-first** [reply]
看了一遍hardhat的文档，感觉讲得挺全了。
